package.json

```diff
@@ -25,6 +25,7 @@
     "babel-loader": "^8.0.6",
     "babel-plugin-formatjs": "^10.2.6",
     "blueimp-md5": "^2.10.0",
+    "chroma-js": "^2.4.2",
```

Comment:
I'm not sure what features we're using of this, but it looks to be a fairly large module compared to all the alternatives, and it's at a size level that is nontrivial for a pretty minor functionality, so we should look at whether a 3x smaller alternative would work instead.

https://bundlephobia.com/package/chroma-js@2.4.2

Reply:
I should add that I'm OK with integrating this as-is and dealing with that as a follow-up if it's nontrivial.

Reply:
Started discussion in https://chat.zulip.org/#narrow/stream/101-design/topic/UI.20redesign.3A.20stream.2Ftopic.20header.20bar/near/1413426